### PR TITLE
Make page size selectable.

### PR DESCRIPTION
**Pull request by andela-onnenanya**
### What does this PR do?
https://bugzilla.mozilla.org/show_bug.cgi?id=1300171

### CODE REVIEW

File: ui/app/js/services/helper_service.js [hunk 1]
```diff
@@ -10,6 +10,11 @@ angular.module("app").factory('Helpers', function() {
       }
       return object;
     },
+    selectPageSize: function($scope){
+      if(+$scope.page_size && +$scope.page_size > 0){
```
Comment (mozbhearsum) [inline ui/app/js/services/helper_service.js]: What are there `+` operators before each of the `$scope.page_size` references?

Comment (andela-onnenanya) [reply ui/app/js/services/helper_service.js]: Oh, an oversight. I should have removed the `unary` operator. It was only meant to convert the input to integer; but that is only required if I used `text input type`.

**Comment by andela-onnenanya** [comment]
Thank you @mozbhearsum, I've effected the changes. Kindly take a look at what I've done.

**Comment by andela-onnenanya** [comment]
@mozbhearsum I've fixed that. Thank you!

**Comment by andela-onnenanya** [comment]
@mozbhearsum I have a question. Making the page size options 20, 50 and All, it implies that the default page size in releases controller should no longer be 10 (since 10 is not in the options). I will have to refactor the test so it passes with the new default page size of 20. Is this the desired behavior or do we have any reason to retain default size of 10 for releases?

**Comment by mozbhearsum** [comment]
We can switch Releases to 20 per page, too - very good question!

**Comment by mozbhearsum** [comment]
Please make sure to merge with upstream to fix the merge conflicts as well - I can't merge the PR until that is done.

**Comment by andela-onnenanya** [comment]
I had to fix the merge conflict here, my app has been clashing since 2 days ago. `/api/rules` returns an empty array.

### Here is the `log`:

```
balrogadmin_1  | [pid: 19|app: 0|req: 44/44] 172.18.0.6 () {50 vars in 1651 bytes} [Fri Oct  6 17:02:57 2017] GET /api/users/current => generated 114 bytes in 15 msecs (HTTP/1.1 200) 8 headers in 290 bytes (1 switches on core 0)
balrogadmin_1  | [2017-10-06 17:02:57,715] ERROR in app: Exception on /rules/columns/product [GET]
balrogadmin_1  | Traceback (most recent call last):
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
balrogadmin_1  |     response = self.full_dispatch_request()
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
balrogadmin_1  |     rv = self.handle_user_exception(e)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
balrogadmin_1  |     reraise(exc_type, exc_value, tb)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
balrogadmin_1  |     rv = self.dispatch_request()
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
balrogadmin_1  |     return self.view_functions[rule.endpoint](**req.view_args)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/connexion/decorators/decorator.py", line 66, in wrapper
balrogadmin_1  |     response = function(request)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/connexion/decorators/validation.py", line 293, in wrapper
balrogadmin_1  |     return function(request)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/connexion/decorators/response.py", line 85, in wrapper
balrogadmin_1  |     response = function(request)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/connexion/decorators/decorator.py", line 42, in wrapper
balrogadmin_1  |     response = function(request)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/connexion/decorators/parameter.py", line 218, in wrapper
balrogadmin_1  |     return function(**kwargs)
balrogadmin_1  |   File "./auslib/web/admin/views/mapper.py", line 53, in single_rule_column_get
balrogadmin_1  |     return SingleRuleColumnView().get(column)
balrogadmin_1  |   File "./auslib/web/admin/views/rules.py", line 179, in get
balrogadmin_1  |     if column not in rules[0].keys():
balrogadmin_1  | IndexError: list index out of range
balrogadmin_1  | [2017-10-06 17:02:57,727] ERROR in base: Caught ISE 500 error.
balrogadmin_1  | Traceback (most recent call last):
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
balrogadmin_1  |     return self.wsgi_app(environ, start_response)
balrogadmin_1  |   File "./auslib/web/admin/base.py", line 47, in __call__
balrogadmin_1  |     return self.app(environ, start_response)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
balrogadmin_1  |     response = self.handle_exception(e)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1547, in handle_exception
balrogadmin_1  |     return self.finalize_request(handler(e), from_error_handler=True)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1630, in finalize_request
balrogadmin_1  |     response = self.make_response(rv)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1740, in make_response
balrogadmin_1  |     rv = self.response_class.force_type(rv, request.environ)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/werkzeug/wrappers.py", line 885, in force_type
balrogadmin_1  |     response = BaseResponse(*_run_wsgi_app(response, environ))
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/werkzeug/wrappers.py", line 57, in _run_wsgi_app
balrogadmin_1  |     return _run_wsgi_app(*args)
balrogadmin_1  |   File "/usr/local/lib/python2.7/site-packages/werkzeug/test.py", line 884, in run_wsgi_app
balrogadmin_1  |     app_rv = app(environ, start_response)
balrogadmin_1  | TypeError: 'exceptions.IndexError' object is not callable
balrogadmin_1  | [pid: 19|app: 0|req: 45/45] 172.18.0.6 () {50 vars in 1675 bytes} [Fri Oct  6 17:02:57 2017] GET /api/rules/columns/product => generated 0 bytes in 51 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)

```

**Comment by andela-onnenanya** [comment]
Ben, I just got this issue solved. I had to remove `mysql` and reinstalled it. I also set the root password. Not sure if it contributed, but I had to switch from `edge` to `stable` docker.

**Pull request merged**